template.yml.erb

```diff
@@ -586,7 +592,7 @@ Resources:
           Then post in #ap-csa-dev.
       ActionsEnabled: true
       AlarmActions:
-          - !If [IsDevCondition, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:javabuilder-dev-test", !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"]
```

Comment:
Just curious, do you know if `javabuilder-dev-test` was setup to notify us?

Reply:
it was setup to go to Jessie's email, which we decided was not ideal so we just silenced it. The alerts will still exist and you can look at the ones on a given instance to see if they have triggered, they just would have any action by default